src/views/ContinuationIn/ContinuationInReviewConfirm.vue

```diff
@@ -252,9 +252,4 @@ h2::before {
   counter-increment: header-counter;
   content: counter(header-counter) '. ';
 }
-
-// override base class text size
-.message-box p {
-  font-size: $px-16;
-}
```

Comment:
Left over from when we had a message box on the amalgamation review page (from which this page was duplicated).